src/rewriter/rw_parser.py

```diff
@@ -204,6 +123,8 @@ def expr(self, allow_comprehension=True):
         app = (pp.Suppress('(') + self.symbol() + pp.OneOrMore(expr) +
                pp.Suppress(')')).setParseAction(self.app_action)
 
+        # let and cond are deprecated. These should be removed
```

Comment:
Shouldn't this be a TODO/FIXME?

Reply:
we need andrew's input on this

Reply:
@ajreynol ->input()

Reply:
You can delete the code for parsing let/cond.

Reply:
Fixed